AuthenticatorBridgeKit/AuthenticatorBridgeItemService.swift

```diff
@@ -201,11 +201,14 @@ public class DefaultAuthenticatorBridgeItemService: AuthenticatorBridgeItemServi
             context: dataStore.persistentContainer.viewContext,
             request: fetchRequest
         )
-        .tryMap { dataItems in
+        .map { dataItems in
             dataItems.compactMap(\.model)
         }
         .asyncTryMap { itemModel in
-            try await self.cryptoService.decryptAuthenticatorItems(itemModel)
+            guard let items = try? await self.cryptoService.decryptAuthenticatorItems(itemModel) else {
```

Comment:
hmm, thinking about this some more. Will this trap other exceptions that the Authenticator app expects to catch and display an error? Should you only catch the keychain error?

Reply:
if we keep going with this approach, then I agree with Victor that this should only catch `KeychainServiceError` and, if possible, the specific error for this situation so it doesn't hide any other potential bugs; which in this case I imagine is `KeychainServiceError.keyNotFound`.

Reply:
I just ran into this bug while working on an Authenticator PR that clarified the reason we're still seeing this when turning off sync:
- The publisher is asking for the list to be decrypted _even when there are no items_ - i.e. it sends `[]` to be decrypted.
- The crypto service is looking for the key first thing, which throws the error. But in the case of an empty list, it never needed the key to decrypt.

So when a user turns off sync (or launches the first time with the feature flag enabled) we _should **not**_ have any items to be decrypted. But because it's looking for the key, it will still fail here.

I'm going to re-work this PR to address the issue in the cypto service, and leave this error handling in place. 👍